tests/bearlib/aspects/ClassTest.py

```diff
@@ -31,3 +33,10 @@ class docs:
                 example = 'Example'
 
         assert not SubAspect.docs.check_consistency()
+        assert issubaspect(SubAspect, RootAspect)
+        assert not issubaspect(Root, RootAspect)
+        assert not issubaspect(RootAspect, RootAspect)
+        with pytest.raises(TypeError):
+            issubaspect('String', SubAspect)
```

Comment:
@Techievena Right! But... :)

**[0]** The assert statement must not be inside the `with` block. Otherwise it won't be reached after the exception occurred.

**[1]** We should not use `unittest`'s `assertEqual` here. Let's keep pythonic `pytest` style and just use `assert ... == ...` :)

**[2]** The `ExceptionInfo` object returned from `pytest.raises` holds the actual exception in its `.value` attribute. It has no `.exception`.

So:

```python
with pytest.raises(TypeError) as exc:
    issubaspect('String', SubAspect)
assert str(exc.value) \
    == "'String' is not an aspectclass or an instance of an aspectclass"
```

@Asnelchristian @shrutishrm512